### PR TITLE
Add support for named tests

### PR DESCRIPTION
**Pull request by ysomad**
`-named` argument for gotests working only from develop branch. For some reason they're not merging it into main considering that a lot of time has passed I think gotests development not active at all atm. Because of that to make this arg work we need to add support for selecting gotests tag to install from.

Problem: hardcoding `@develop` tag in `urls` table works but if trying to get tag from gotests config there is always default value imported only.

### CODE REVIEW

**Review by skydiver1980 — APPROVED** [review]

**Comment by ysomad** [comment]
Before merging we need to fix the issue with a tag which exports as default value only

**Comment by skydiver1980** [comment]
Please fix and I'll approve 5hanks

**Comment by ysomad** [comment]
Didn't find any good solution which allow reference current user config at any time while running `:GoInstallDeps`. I think manually installing gotests from develop in on plugin build is good enough and not overwhelm current codebase.

@olexsmir Think we good to go

**Comment by ysomad** [comment]
@olexsmir any chance it'll be reviewed?

File: README.md [hunk 1]
```diff
@@ -20,6 +20,27 @@ use {
...
 ```
 
+Lazy.nvim:
+
+```lua
+return {
+  "olexsmir/gopher.nvim",
+  build = function ()
+    vim.cmd(":GoInstallDeps")
+    -- make named tests works
```
Comment (olexsmir) [inline README.md:30]: i think it would be better to mark this part of the config as optional and required only if the user wanna use the `named` option

**Comment by olexsmir** [comment]
sorry for the long time that it took me to review the PR. thank you so much PR

**Comment by olexsmir** [comment]
thank you for the PR, and sorry for the long wait

**Pull request merged**